m/mage4.md

```diff
@@ -0,0 +1,9 @@
+# Mage 4
+
+![Mage4](MAGE-4-300x300.gif)
+
+>This artist started to sing in *1992* and became famous *2008*. There are eight people in *Mage4*: **Helien, Lita, Mika, Tins, Elise, Solohery,Zeze and Bema**.They are living in ***Alarobia Antananarivo***.
+
+**Genre:** Soft rock
+
+**Artist homepage:** [Mage4](https://www.google.com/search?rlz=1C1PQHB_enMG885MG885&sxsrf=ACYBGNSJDf0UKqInyHSI5e9QoG7m1fdgfQ%3A1580296132920&ei=xGcxXo_nN5P2gAbtzrCwAg&q=mage+4&oq=mage+4&gs_l=psy-ab.3..35i39l3j0l2j0i20i263j0j0i203l2j0.6746.88437..91440...3.0..0.518.15283.3-23j13j3......0....1..gws-wiz.......35i304i39j0i13j0i7i30j0i22i30j0i22i10i30.RNYmtvp6V7k&ved=0ahUKEwiP2ZHt1ajnAhUTO8AKHW0nDCYQ4dUDCAs&uact=5)
```

Comment:
Try to find their Facebook or homepage and use that instead of a google search results page.